packages/web/src/pages/sign-in-page/SignInPage.tsx

```diff
@@ -37,6 +38,8 @@ import { SignInWithMetaMaskButton } from './SignInWithMetaMaskButton'
 
 const SignInSchema = toFormikValidationSchema(signInSchema)
 
+const smallDesktopWindowHeight = 900
```

Comment:
900 seems like a good number to me
 ![](https://media1.tenor.com/m/-JIgHIMq1u8AAAAC/cool.gif)